src/test/java/tech/sirwellington/alchemy/kotlin/extensions/MapsPlusTest.kt

```diff
@@ -85,4 +85,22 @@ class MapsPlusTest
         assertThat(map.size, equalTo(sizeBefore))
         assertThat(map, equalTo(original))
     }
-}
\ No newline at end of file
+
+    @Test
+    fun testAnyEntry()
+    {
+        val entry = map.anyEntry!!
+
+        assertTrue(map.containsKey(entry.key))
+        assertTrue(map.containsValue(entry.value))
```

Comment:
Add one more assertion to make sure the key-value relationship itself exists.

Reply:
Added the assertion, thanks!